### PR TITLE
hackathon feature added

### PR DESCRIPTION
**Pull request by umangahuja1**
Review.

I have tested this feature.

There is one little thing to add more i.e Install phantomJS. Can anyone add those commands as I have installed it locally?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,7 @@ env/
 .idea/
 MANIFEST.in
 setup.cfg
-setup.py
\ No newline at end of file
+setup.pyjarviscli/CmdInterpreter.py
```
Comment (appi147) [inline .gitignore]: CmdInterpreter and hackathon py files are here by mistake

**Comment by appi147** [comment]
I have added script for phantomjs

**Comment by appi147** [comment]
Please test it if it works

**Comment by umangahuja1** [comment]
Review changes. Working on my system with necessary changes

**Comment by appi147** [comment]
appi$~ pip install phantomjs
Collecting phantomjs
  Could not find a version that satisfies the requirement phantomjs (from versions: )
No matching distribution found for phantomjs

**Comment by appi147** [comment]
You sure it is on pypi? [PyPi](https://pypi.python.org/pypi?%3Aaction=search&term=phantomjs&submit=search)

**Comment by appi147** [comment]
Nice work. I have tested it

**Comment by appi147** [comment]
Could be more cool if some animations were implemented while fetching list

**Comment by sukeesh** [comment]
@appi147 @umangahuja1 

Sorry for late reply.
This includes all hackathons. ( http://clist.by/ )
They also provide an API. Let me know your thoughts on this.

**Pull request merged**